versioned_docs/version-4.x/bottom-tab-navigator.md

```diff
@@ -29,6 +29,7 @@ The route configs object is a mapping from route name to a route config, which t
 ## BottomTabNavigatorConfig
 
 - `initialRouteName` - The routeName for the initial tab route when first loading.
+- `detachInactiveScreens` - Boolean used to indicate whether inactive screens should be detached from the view hierarchy to save memory. Make sure to call `enableScreens` from [react-native-screens](https://github.com/software-mansion/react-native-screens) to make it work. Defaults to `true`.
```

Comment:
Place navigator related configs after the router related configs (after `backBehavior`)